docs/tutorial/gatsby-image-tutorial

```diff
@@ -0,0 +1,275 @@
+          }
+        }
+      }
+    }
+  `)
+
+  return <Img fixed={data.file.childImageSharp.fixed} />
+}
+```
+
+Instead of a query constant and data that references the result, you can use a static query hook directly in the JXS code and then reference it. Note that the query language didn’t change and neither did the `Img` tag syntax, the only change was the location of the query and the usage of the `useStaticQuery` function to wrap it.
+
+### Multiple Queries and Aliasing
+
+The last use case you may come across is how to handle a situation where you have multiple queries in the same file/page.
+
+This example is attempting to query for all the data in `speaking.yaml` and the direct file query in our first example. In order to do this you want to use aliasing.
+
+The first thing to know is that an alias is assigning a name to a query. Below is a simple example.
+
+```jsx
+talks: allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+When you do that, you’ve changed the reference to that object in your JSX. While it was previously referenced as
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
+```
+
+giving it an alias does not add a level of complexity to the response object, it just replaces it. So you end up with the same structure referenced as
+
+```jsx
+{data.talks.edges.map(({ node }) => ())
+```
+
+The top-level object name of data is implicit. This is important because when you conduct multiple queries as part of a single component `Gatsby` still passes the entire result to the component.
+
+```jsx
+const SpeakingPage = ({ data}) => {})
+```
+
+everything else gets referenced from that top-level return
+name.
+
+With that understanding, you can combine two queries and use aliasing to distinguish between them.
+
+```jsx
+{
+    allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+    banner: file(relativePath: { eq: "headers/default.jpg" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+  }
+}
+```
+
+Notice that this example uses aliasing for one query and not the other. This is allowed; there is no requirement that all your queries use aliasing. In this case, the JSX would look like this to access the `speaking.yaml` content.
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
+```
+
+And then like this to access the image using the alias name `banner`.
+
+```jsx
+<Img fluid={data.banner.childImageSharp.fluid} />
+```
+
```

Comment:
```suggestion

These examples should handle a fair number of use cases. A couple bonus things:
```
Moving this above the aspect ratio heading, as it might flow a little better to set up the next two examples.